docs/operating-systems/course-tools.md

```diff
@@ -202,35 +201,52 @@ You want your output to say you have the clang version.
           
       clean:
           rm hello.o hello.exe```
+  
 9. Navigate to the directory in your terminal, enter make and see the program run!
 
 ### Docker
-- Navigate to https://docs.docker.com/desktop/install/windows-install/, get the installer, and follow the instructions.
+- Navigate to https://docs.docker.com/desktop/install/windows-install/ to install Docker, and follow the instructions.
 
 ### Python 3.10
-1. Go to the Microsoft Store app, search ```Python 3.10``` and install.
-2. Can check this worked by using the ```python --version``` command.
+Go to the Microsoft Store app, and install: 
```

Comment:
This is all being shown on one line. Maybe it should be broken up into multiple?